test/unit_Polytope.jl

```diff
@@ -132,6 +132,14 @@ for N in [Float64, Rational{Int}, Float32]
         # list of constraints of a VPolytope; calculates tohrep
         @test ispermutation(constraints_list(V), constraints_list(tohrep(V)))
     end
+
+    # copy (see #1002)
+    p, q = [1], [2]
+    P = VPolytope([p, q])
+    Pcopy = copy(P)
+    p[1] = 5
+    # test that Pcopy is independent of P ( = deepcopy)
+    @test Pcopy.vertices[1] == [1]
```

Comment:
That test does not use the numeric type `N`. So we execute the same test three times.